core/src/main/scala/quasar/destination/avalanche/package.scala

```diff
@@ -115,27 +115,31 @@ package object avalanche {
         .queryWithLogHandler[Int](logHandler)
     }
 
+    def grantAllTableUpdate: Update0 =
+      (fr"GRANT ALL PRIVILEGES ON TABLE" ++ Fragment.const(tableName) ++ fr"TO group dbadmingrp")
+        .updateWithLogHandler(logHandler)
+
     writeMode match {
       case WriteMode.Replace =>
-        dropTableUpdate.run >> createTableUpdate.run
+        dropTableUpdate.run >> createTableUpdate.run >> grantAllTableUpdate.run
```

Comment:
Could DRY these up with something like `val initializeTable = createTableUpdate.run >> grantAlltableUpdate.run`

Reply:
Right yeah, I was hesitating about this too